### ISSUE TITLE
Add support of non-linear operations

### ISSUE THREAD

**Issue by Ther-nullptr** [MrYxJ/calculate-flops.pytorch#34]
Hello, it seems that the calculator cannot record the operation num of non-linear operations like layernorm and softmax, can you support it?
